tests/test_django.py

```diff
@@ -20,15 +21,16 @@ def test_parse_error():
         get_template("parse_error.txt", using="rusty")
 
     template_dir = Path("tests/templates").absolute()
+    path_separator = '/' if os.name != 'nt' else '\\'
```

Comment:
We can use [`os.sep`](https://docs.python.org/3/library/os.html#os.sep) instead:

```suggestion
```

Reply:
TIL! They have everything in the stdlib, what will they think of next?!